Library/AddOn.cs

```diff
@@ -29,7 +29,18 @@ public enum Type
         public DateTime UpdatedAt { get; private set; }
         public Adjustment.RevenueSchedule? RevenueScheduleType { get; set; }
         public string ItemState { get; set; }
-        public string TierType { get; set; }
+        private string _tierType;
+        public string TierType { 
+          get
+          {
+            if (_tierType == null)
```

Comment:
I don't think you need to set the _tierType in the getter.

```
get
{
if (_tierType == null) return "flat";
return _tierType;
}
```

Reply:
Or maybe just:

```
get
{
  return (_tierType == null) ? "flat" : _tierType;
} 
```

Reply:
Much more eloquent!